webrtc.html

```diff
@@ -7669,11 +7721,20 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     <p>The WebRTC specification provides no user prompts or chrome
     indicators for communication; it assumes that once the Web page
     has been allowed to access media, it is free to share that media
-    with other entities as it chooses.</p>
-
-    <p>A mechanism, <code><a>peerIdentity</a></code>, is provided that gives
-    Javascript the option of requesting media that the same javascript
-    cannot access, but can only be sent to certain other entities.</p>
+    with other entities as it chooses. Peer-to-peer exchanges of data
+    view WebRTC datachannels can thus occur without any user explicit
+    consent or involvement, similarly as a server-mediated exchange
+    (e.g. via Web Sockets) could occur without user involvement.</p>
+
+    <p>The <code><a>peerIdentity</a></code> mechanism, used to establish
+    communications in which the application itself cannot intercept,
```

Comment:
Grammar: "in which" -> "which". But the whole part between the commas should go; the identity mechanism is not used only for that.